### PR TITLE
Update de_de.lang

### PR DESCRIPTION
**Pull request by Sven792**

### CODE REVIEW

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 1]
```diff
@@ -15,63 +15,63 @@ enchantment.openblocks.laststand=Letzte Rettung
 enchantment.openblocks.flimflam=Blödsinn
 
 openblocks.gui.blocks=Blöcke
-openblocks.gui.items=Items
+openblocks.gui.items=Gegenstände
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: See [this comment](https://github.com/mezz/JustEnoughItems/pull/1438#pullrequestreview-186946544).

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 2]
```diff
@@ -15,63 +15,63 @@ enchantment.openblocks.laststand=Letzte Rettung
...
 openblocks.gui.vacuumhopper=Vakuumtrichter
 openblocks.gui.bigbutton=Großer Knopf
 openblocks.gui.xpbottler=XP-Abfüllgerät
-openblocks.gui.autoanvil=Auto-Amboss
-openblocks.gui.autoeject=Auto-Auswerfen
-openblocks.gui.autoextract=Auto-Extrahieren
-openblocks.gui.autodrink=Auto-Trinken
-openblocks.gui.autoenchantmenttable=Auto-Zaubertisch
+openblocks.gui.autoanvil=Automatischer Amboss
+openblocks.gui.autoeject=Automatischer Auswerfer
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: This... doesn't fit in the GUI.

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 3]
```diff
@@ -15,63 +15,63 @@ enchantment.openblocks.laststand=Letzte Rettung
...
+openblocks.gui.autoextract=Automatischer Extrahierer
+openblocks.gui.autodrink=Automatisches Trinkgerät
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: Are you aware these are terms in a UI?

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 4]
```diff
@@ -15,63 +15,63 @@ enchantment.openblocks.laststand=Letzte Rettung
...
 openblocks.gui.blockplacer=Blockplatzierer
-openblocks.gui.paintmixer=Farbmixer
+openblocks.gui.paintmixer=Farbenmischer
 openblocks.gui.drawingtable=Zeichentisch
-openblocks.gui.digitalfuse=Digitalzünder
-openblocks.gui.xp_outputs=XP-Abgabe
-openblocks.gui.item_outputs=Itemabgabe
-openblocks.gui.limit=Limit
+openblocks.gui.digitalfuse=Digitaler Zünder
+openblocks.gui.xp_outputs=Erfahrungspunkten-Abgabe
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: Wrong grammar.

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 5]
```diff
@@ -15,63 +15,63 @@ enchantment.openblocks.laststand=Letzte Rettung
...
-openblocks.gui.save_folder=Sichere Ordner
+openblocks.gui.save_folder=Ordner sichern
 
 openblocks.gui.drawingtable.stencils=Schablonen
 openblocks.gui.drawingtable.glyphs=Glyphen
 openblocks.gui.drawingtable.print=Druck
 
 openblocks.gui.welcome.title=Willkommen!
-openblocks.gui.welcome.content=OpenBlocks ist eine neue Open-Source-Mod, die verschiedene Ideen in Minecraft einbringt, inklusive Gleitschirme, Aufzüge, Tanks, Gräber, Farben, Kräne, flüssige XP, Kartographen, Trophäen und mehr!
+openblocks.gui.welcome.content=OpenBlocks ist eine neue freie Modifikation, die verschiedene Ideen in Minecraft einbringt, inklusive Gleitschirme, Aufzüge, Tanks, Gräber, Farben, Kräne, flüssige Erfahrungspunkte, Kartographen, Trophäen und mehr!
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: It should stay "Mod".

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 6]
```diff
@@ -15,63 +15,63 @@ enchantment.openblocks.laststand=Letzte Rettung
...
 
-openblocks.gui.restore_inv.title=Inventar-Backup
-openblocks.gui.restore_inv.content=Jeder kann einen Fehler machen, aber mit dem §L/ob_inventory§R-Befehl kannst du einige von ihnen wieder beheben. Dieser Befehl kann zum Wiederherstellen des Inventars genutzt werden. Ein Backup wird erstellt wenn:\n - der Spieler stirbt\n - ein Grabstein gespawnt wird \n - der Spieler den Befehl §L/ob_inventory store§R eingibt\n\nTo restore an inventory you need the name of the backup. Gehen in den unten genannten Ordner oder den saves-Ordner bei Servern und such nach Dateien mit dem Namen §Linventory-*.dat§R. Dann stelle das Inventar wieder her mit §L/ob_inventory restore <player> <id>§R - wobei id der volle Dateiname ist, oder nur der mittlere Teil ohne §Linventory-§R and §L.dat§R. Du kannst Tabvervollständigung benutzen. Backups sind normal NBT-Dateien und können mit einem entsprechenden Editor geöffnet werden.
+openblocks.gui.restore_inv.title=Inventar-Sicherung
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: Not sure this is good to translate.

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 7]
```diff
@@ -87,7 +87,7 @@ openblocks.misc.mode.inverted_block=Invertierter Block
 openblocks.misc.mode.inverted_panel=Invertierte Platte
 openblocks.misc.mode.inverted_half_panel=Invertierte Halbhohe Platte
 openblocks.misc.mode.inverted_stairs=Invertierte Treppe
-openblocks.misc.structure_pos=Debug: Struktur %s gefunden bei (%d,%d,%d)
+openblocks.misc.structure_pos=Fehlersuche: Struktur %s gefunden bei (%d,%d,%d)
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: The word "Debug" shouldn't be translated.

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 8]
```diff
@@ -98,8 +98,8 @@ openblocks.misc.total_blocks=Insgesamt gezählte Blöcke: %d
 openblocks.misc.get_witched=Werde verhext!
 openblocks.misc.page=Seite %d von %d
 openblocks.misc.oh_no_ceiling=Du kannst hier nicht schlafen. Die Decke beunruhigt dich zu sehr...
-openblocks.misc.oh_no_ground=Auf DEM Ding willst du schlafen?!
-openblocks.misc.sleeping_bag_broken=Item inaktiv wegen fehlerhafter Initialisierung
+openblocks.misc.oh_no_ground=Auf DEM DING willst du schlafen?!
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: Wrong stress.

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 9]
```diff
@@ -222,91 +222,91 @@ item.openblocks.sponge_on_a_stick.name=Schwamm
...
 
 item.openblocks.epic_eraser.name=Geschichtsveränderndes Radiergummi
-item.openblocks.epic_eraser.description=Jedes Item erzählt eine Geschichte...\n\nIst das nicht eigentlich saublöd?\n Mit diesem Item kannst du jegliche Spuren epischer Abenteuer, die dein Kram im Laufe der Jahrtausende erlebt hat, vollständig vernichten. Dann wird auch keiner Fragen stellen, wenn du ihn auf dem Schwarzmarkt vertickst.
+item.openblocks.epic_eraser.description=Jeder Gegenstand erzählt eine Geschichte...\n\nIst das nicht eigentlich saublöd?\n Mit diesen Gegenstand kannst du jegliche Spuren epischer Abenteuer, die dein Kram im Laufe der Jahrtausende erlebt hat, vollständig vernichten. Dann wird auch keiner Fragen stellen, wenn du ihn auf dem Schwarzmarkt vertickst.
 
-tile.openblocks.cannon.name=Item-Kanone
-tile.openblocks.cannon.description=Wenn ein Redstone-Signal anliegt, wird die Item-Kannone Items, die es in einem benachbarten Inventar findet, schießen. Benutze den 'Zeiger', um die Kannone auszurichten.
+tile.openblocks.cannon.name=Gegenstands-Kanone
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: This just sounds horrible.

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 10]
```diff
@@ -222,91 +222,91 @@ item.openblocks.sponge_on_a_stick.name=Schwamm
...
+tile.openblocks.cannon.description=Wenn ein Redstone-Signal anliegt, wird die Gegenstands-Kannone Gegenstände, die es in einem benachbarten Inventar findet, schießen. Benutze den 'Zeiger', um die Kannone auszurichten.
 
 tile.openblocks.rope_ladder.name=Strickleiter
 tile.openblocks.rope_ladder.description=Wenn du eine Strickleiter platzierst, wird sie sich bis zum Boden ausrollen.
 
 tile.openblocks.sprinkler.name=Rasensprenger
-tile.openblocks.sprinkler.description=Der Rasensprenger hilft dir, Pflanzen bis zu 4 Blöcke weit weg zu düngen. Platziere ihn einfach über einem OpenBlocks-Tank, der mit Wasser gefüllt ist! Du kannst auch Knochenmehl für noch schnelleres Wachstum hinzufügen.\nEr zählt auch als Wasser-Quellblock!
+tile.openblocks.sprinkler.description=Der Rasensprenger hilft dir, Pflanzen bis zu 4 Blöcke weit weg, zu düngen. Platziere ihn einfach über einem OpenBlocks-Tank, der mit Wasser gefüllt ist! Du kannst auch Knochenmehl für noch schnelleres Wachstum hinzufügen.\nEr zählt auch als Wasser-Quellblock!
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: That comma does not belong there.

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 11]
```diff
@@ -222,91 +222,91 @@ item.openblocks.sponge_on_a_stick.name=Schwamm
...
 
-tile.openblocks.paint_mixer.name=Farbmixer
-tile.openblocks.paint_mixer.description=Der Farbmixer kann jede Farbe, die du willst, zusammenmixen! Gib entweder einen Farbeimer oder einen Eimer Milch in den oberen Slot, dann Türkisen, Magentafarbenen, Gelben und Schwarzen Farbstoff in die anderen vier Slots und klicke auf Mixen!
+tile.openblocks.paint_mixer.name=Farbenmischer
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: Wrong grammar.

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 12]
```diff
@@ -222,91 +222,91 @@ item.openblocks.sponge_on_a_stick.name=Schwamm
...
 tile.openblocks.target.name=Zielscheibe
-tile.openblocks.target.description=Die Zielscheibe gibt ein Redstone-Signal mit einer Stärke abhängig von der Genauigkeit des Schützen aus. Gib einer Zielscheibe ein Redstone-Signal, um sie auklappen zu lassen!
+tile.openblocks.target.description=Die Zielscheibe gibt ein Redstone-Signal mit einer Stärke abhängig von der Genauigkeit des Schützen aus. Gib einer Zielscheibe ein Redstone-Signal, um sie ausklappen zu lassen!
 
 tile.openblocks.path.name=Pfad
 tile.openblocks.path.description=Der Pfad macht überhaupt nichts. Er sieht einfach nur gut aus!
 
-tile.openblocks.item_dropper.name=Itemspender
-tile.openblocks.item_dropper.description=Der Itemspender ist dem Vanilla-Spender ähnlich, außer dass er Items direkt in einer geraden Linie nach unten wirft.
+tile.openblocks.item_dropper.name=Gegenstandspender
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: Missing `s` and it sounds bad anyway.

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 13]
```diff
@@ -222,91 +222,91 @@ item.openblocks.sponge_on_a_stick.name=Schwamm
...
+tile.openblocks.auto_anvil.description=Der Automatischer Amboss funktioniert ähnlich wie ein Vanilla-Amboss, außer dass er mit flüssigen Erfahrungspunkten arbeitet und komplett automatisierbar ist.\n\nBenutze die Tabs auf der rechten Seite der Oberfläche, um auszuwählen, welche Seiten des Ambosses für Ein-/Ausgabe von Gegenstände oder Flüssigkeiten vorgesehen sind. Rechts-klicke und ziehe den Amboss im Tab, um ihn zu drehen und andere Seiten auszuwählen!
 
 tile.openblocks.drawing_table.name=Zeichentisch
 tile.openblocks.drawing_table.description=Der Zeichentisch wird gebraucht, um Schablonen herzustellen, die zum Zeichnen gebraucht werden. Gib eine unbearbeitete Schablone in den Zeichentisch und benutze die Pfeile, um die gewünschte Schablone auszuwählen!
 
 tile.openblocks.grave.name=Grab
 tile.openblocks.trophy.name=Trophäe
 tile.openblocks.trophy.entity.name=%s Trophäe
-tile.openblocks.canvasglass.name=Glasleinwand
+tile.openblocks.canvasglass.name=Glas-Leinwand
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: I prefer how it was before. Sounds better.

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 14]
```diff
@@ -222,91 +222,91 @@ item.openblocks.sponge_on_a_stick.name=Schwamm
...
+tile.openblocks.radio.description=Das Radio erlaubt es dir, Internetradio zu spielen! Zunächst brauchst du Wellenkristalle, die du in Truhen aus Verliesen findest oder von einem bestimmten Dorfbewohner bekommst. Platziere den Kristall ins Radio und gib diesem ein Redstone-Signal, um es anzuschalten. Die Stärke des Redstone-Signals bestimmt die Lautstärke.\n\nIn der Konfigurations-Datei des Servers kannst du konfigurieren, welche Radiosender verfügbar sind.
 
-tile.openblocks.xp_shower.name=XP-Dusche
-tile.openblocks.xp_shower.description=Bringe eine XP-Dusche an einen Tank voller XP-Saft an und sie wird XP-Orbs ausschütten. Diese Orbs können von einem XP-Gitter oder einer MFR-Kloake aufgesammelt werden.
+tile.openblocks.xp_shower.name=Erfahrungspunkte-Dusche
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: Why this change? The former translation was a lot nicer.

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 15]
```diff
@@ -384,17 +384,17 @@ item.openblocks.tasty_clay.name=Leckerer Lehm
 item.openblocks.golden_eye.name=Goldenes Auge
 item.openblocks.pointer.name=Zeiger
 item.openblocks.tuned_crystal.name=Wellenkristall
-item.openblocks.pedometer.name=Pedometer
+item.openblocks.pedometer.name=Schrittzähler
 
-item.openblocks.wallpaper.name=Poster
+item.openblocks.wallpaper.name=Plakat
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: Not the same thing.

Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: Oh, it's just a synonym...

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 2 above)
Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: "Autom." is the solution. "Auto" also means "car" in german. :D

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 3 above)
Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: "Autom." is the solution.

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 2 above)
Comment (Vexatos) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: Yes, short for "Automobil", same meaning as "Auto" here.

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 1 above)
Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: I know, we are speaking Denglish today.

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 6 above)
Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: I know "Sicherung" is sadly a german word.

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 10 above)
Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: The first time I read it: "weg zu düngen" xD

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 14 above)
Comment (Vexatos) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: Not everything has to be translated literally, you know.

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 5 above)
Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: "Mod" is in the dictionary, but it didn't have the same meaning as modification :D 
https://www.duden.de/rechtschreibung/Mod

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 14 above)
Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: No :P

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 5 above)
Comment (Vexatos) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: But... It was obvious in this context. Are you not translating without context in mind?

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 6 above)
Comment (Vexatos) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: So what? There is no need to translate the word "Backup".

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 10 above)
Comment (Vexatos) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: Doesn't change the fact that that comma is placed wrong.

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 16]
```diff
@@ -222,72 +222,72 @@ item.openblocks.sponge_on_a_stick.name=Schwamm
...
 tile.openblocks.target.name=Zielscheibe
-tile.openblocks.target.description=Die Zielscheibe gibt ein Redstone-Signal mit einer Stärke abhängig von der Genauigkeit des Schützen aus. Gib einer Zielscheibe ein Redstone-Signal, um sie auklappen zu lassen!
+tile.openblocks.target.description=Die Zielscheibe gibt ein Redstone-Signal mit einer Stärke abhängig von der Genauigkeit des Schützen aus. Gib einer Zielscheibe ein Redstone-Signal, um sie ausklappen zu lassen!
 
 tile.openblocks.path.name=Pfad
 tile.openblocks.path.description=Der Pfad macht überhaupt nichts. Er sieht einfach nur gut aus!
 
-tile.openblocks.item_dropper.name=Itemspender
-tile.openblocks.item_dropper.description=Der Itemspender ist dem Vanilla-Spender ähnlich, außer dass er Items direkt in einer geraden Linie nach unten wirft.
+tile.openblocks.item_dropper.name=Sachenspender
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: Uhhh.... No.

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 17]
```diff
@@ -222,72 +222,72 @@ item.openblocks.sponge_on_a_stick.name=Schwamm
...
 
 tile.openblocks.vacuum_hopper.name=Vakuumtrichter
-tile.openblocks.vacuum_hopper.description=Der Vakuumtrichter saugt Items und XP-Kugeln um sich herum auf. Du kannst die Tabs auf der rechten Seite des Interfaces verwenden, um auszuwählen, zu welchen Seiten etwas ausgegeben werden soll.
+tile.openblocks.vacuum_hopper.description=Der Vakuumtrichter saugt Gegenstände und Erfahrungs-Kugeln um sich herum auf. Du kannst die Tabs auf der rechten Seite der Oberfläche verwenden, um auszuwählen, zu welchen Seiten etwas ausgegeben werden soll.
 
-tile.openblocks.village_highlighter.name=Dorf-Highlighter
-tile.openblocks.village_highlighter.description=Platziere den Dorf-Highlighter in einem Dorf und gib ihm ein Redstone-Signal. Er wird den Umriss des Dorfes darstellen und eine Box im Eisengolem-Spawnbereich zeichnen. Nützlich, wenn du sichergehen willst, dass Dörfer nicht verschmelzen, oder wenn du eine Eisengolem-Farm baust!
+tile.openblocks.village_highlighter.name=Dorf-Markierer
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: That just is a horrible translation. Just don't.

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 18]
```diff
@@ -222,72 +222,72 @@ item.openblocks.sponge_on_a_stick.name=Schwamm
 item.openblocks.sponge_on_a_stick.description=Der Schwamm am Stiel ist ein Werkzeug zum Enfernen von Flüssigkeiten. Funktioniert wie der Schwamm, ist aber mobil und wird mit der Zeit aufgebraucht.
 
 item.openblocks.epic_eraser.name=Geschichtsveränderndes Radiergummi
-item.openblocks.epic_eraser.description=Jedes Item erzählt eine Geschichte...\n\nIst das nicht eigentlich saublöd?\n Mit diesem Item kannst du jegliche Spuren epischer Abenteuer, die dein Kram im Laufe der Jahrtausende erlebt hat, vollständig vernichten. Dann wird auch keiner Fragen stellen, wenn du ihn auf dem Schwarzmarkt vertickst.
+item.openblocks.epic_eraser.description=Jeder Gegenstand erzählt eine Geschichte...\n\nIst das nicht eigentlich saublöd?\n Mit diesen Gegenstand kannst du jegliche Spuren epischer Abenteuer, die dein Kram im Laufe der Jahrtausende erlebt hat, vollständig vernichten. Dann wird auch keiner Fragen stellen, wenn du ihn auf dem Schwarzmarkt vertickst.
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: Grammatically wrong, apart from this change being bad in general.

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 19]
```diff
@@ -303,10 +303,10 @@ tile.openblocks.sky.normal.description=Bringe das Außen nach innen mit dem neue
...
+tile.openblocks.radio.description=Das Radio erlaubt es dir, Internetradio zu spielen! Zunächst brauchst du Wellenkristalle, die du in Truhen aus Verliesen findest oder von einem bestimmten Dorfbewohner bekommst. Platziere den Kristall ins Radio und gib diesem ein Redstone-Signal, um es anzuschalten. Die Stärke des Redstone-Signals bestimmt die Lautstärke.\n\nIn der Konfigurations-Datei des Servers kannst du konfigurieren, welche Radiosender verfügbar sind.
 
 tile.openblocks.xp_shower.name=XP-Dusche
-tile.openblocks.xp_shower.description=Bringe eine XP-Dusche an einen Tank voller XP-Saft an und sie wird XP-Orbs ausschütten. Diese Orbs können von einem XP-Gitter oder einer MFR-Kloake aufgesammelt werden.
+tile.openblocks.xp_shower.description=Bringe eine XP-Dusche an einen Tank voller Erfahrungs-Saft an und sie wird Erfahrungs-Kugeln ausschütten. Diese Kugeln können von einem Erfahrungspunkte-Gitter oder einer MFR-Kloake aufgesammelt werden.
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: I really dislike that you translated XP to... this.

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 6 above)
Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: But "Sicherung" is german and "Backup" is english.

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 5 above)
Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: Is there a problem if we use the word "Modifikation"?

Comment (Vexatos) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: It is inconsistent, and mostly just sounds bad. "Mod" is a known word and should be used.

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 17 above)
Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: Do you have an other idea?

Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: No "highlighter", it is not german.

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 20]
```diff
@@ -184,20 +184,20 @@ openblocks.misc.hidden_lore=§a§oHalte ALT gedrückt für die Geschichte§r
...
 
 openblocks.misc.dropper.speed=Geschwindigkeit
-openblocks.misc.dropper.use_redstone=Hängt von Redstone-Signalstärke ab
+openblocks.misc.dropper.use_redstone=Hängt von der Redstone-Signalstärke ab
 
 openblocks.turtle.magnet=Magnet
 
 tile.openblocks.elevator.name=Fahrstuhl
-tile.openblocks.elevator.description=Platziere einen Fahrstuhl drei oder mehr Blöcke oberhalb eines anderen ohne Blöcke dazwischen und du wirst fähig sein, entweder zu 'springen', um dich aufwärts zu bewegen oder zu 'kriechen', um dich abwärts zu bewegen. Fahrstühle können gefärbt werden, indem man mit einem Farbstoff in der Hand rechtsklickt, jedoch kannst du dich nur zwischen Fahrstühlen in der selben Farbe teleportieren.
+tile.openblocks.elevator.description=Platziere einen Fahrstuhl drei oder mehr Blöcke oberhalb eines anderen ohne Blöcke dazwischen und du wirst fähig sein, entweder zu 'springen', um dich aufwärts zu bewegen oder zu 'kriechen', um dich abwärts zu bewegen. Fahrstühle können gefärbt werden, indem man mit einem Farbstoff in der Hand rechtsklickt. Wie auch immer, du kannst dich jedoch nur zwischen Fahrstühlen in der selben Farbe teleportieren.
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: You have tried translating this literally. This is not how that works.

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 21]
```diff
@@ -99,7 +99,7 @@ openblocks.misc.get_witched=Werde verhext!
 openblocks.misc.page=Seite %d von %d
 openblocks.misc.oh_no_ceiling=Du kannst hier nicht schlafen. Die Decke beunruhigt dich zu sehr...
 openblocks.misc.oh_no_ground=Auf DEM Ding willst du schlafen?!
-openblocks.misc.sleeping_bag_broken=Item inaktiv wegen fehlerhafter Initialisierung
+openblocks.misc.sleeping_bag_broken=Gegenstand inaktiv wegen fehlerhafter Initialisierung
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: Stop trying to translate "Item".

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 22]
```diff
@@ -393,8 +393,8 @@ itemGroup.tabOpenBlocks=OpenBlocks
 entity.openblocks.luggage.name=Reisegepäck
 entity.openblocks.cartographer.name=Kartograph
 entity.openblocks.block.name=Schwebender Block
-entity.openblocks.mini_me.name=Mini-Me
+entity.openblocks.mini_me.name=Kleines Ich
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: This is a direct reference and probably shouldn't be translated.

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 5 above)
Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: But no, it is not correct.

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 22 above)
Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: I didn't know.

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 21 above)
Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: However :P

File: src/main/resources/assets/openblocks/lang/de_de.lang [hunk 23]
```diff
@@ -222,72 +222,72 @@ item.openblocks.sponge_on_a_stick.name=Schwamm
...
+tile.openblocks.cannon.description=Wenn ein Redstone-Signal anliegt, wird die Item-Kannone Gegenstände, die es in einem benachbarten Inventar findet, schießen. Benutze den 'Zeiger', um die Kannone auszurichten.
 
 tile.openblocks.rope_ladder.name=Strickleiter
 tile.openblocks.rope_ladder.description=Wenn du eine Strickleiter platzierst, wird sie sich bis zum Boden ausrollen.
 
 tile.openblocks.sprinkler.name=Rasensprenger
-tile.openblocks.sprinkler.description=Der Rasensprenger hilft dir, Pflanzen bis zu 4 Blöcke weit weg zu düngen. Platziere ihn einfach über einem OpenBlocks-Tank, der mit Wasser gefüllt ist! Du kannst auch Knochenmehl für noch schnelleres Wachstum hinzufügen.\nEr zählt auch als Wasser-Quellblock!
+tile.openblocks.sprinkler.description=Der Rasensprenger hilft dir die Pflanzen zu düngen und hat einen Radius von bis zu 4 Blöcken. Platziere ihn einfach über einem OpenBlocks-Tank, der mit Wasser gefüllt ist! Du kannst auch Knochenmehl für noch schnelleres Wachstum hinzufügen.\nEr zählt auch als Wasser-Quellblock!
```
Comment (Vexatos) [inline src/main/resources/assets/openblocks/lang/de_de.lang]: You removed a comma too many.
```suggestion
tile.openblocks.sprinkler.description=Der Rasensprenger hilft dir, Pflanzen zu düngen, und hat einen Radius von bis zu 4 Blöcken. Platziere ihn einfach über einem OpenBlocks-Tank, der mit Wasser gefüllt ist! Du kannst auch Knochenmehl für noch schnelleres Wachstum hinzufügen.\nEr zählt auch als Wasser-Quellblock!
```

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 21 above)
Comment (Vexatos) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: Yes, that is not what "However" means...

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 19 above)
Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: I change it, but what is about "Erfahrungs-Kugeln"?

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 17 above)
Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: Ok

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 19 above)
Comment (Vexatos) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: Minecraft's own translation for it is "Erfahrungskugel". I'd just remove the hyphen, same with the "Saft".

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 5 above)
Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: Still not, however.

File: src/main/resources/assets/openblocks/lang/de_de.lang (hunk 19 above)
Comment (Sven792) [reply src/main/resources/assets/openblocks/lang/de_de.lang]: Ok then so.

**Comment by Sven792** [comment]
@Vexatos ?

**Comment by Vexatos** [comment]
You still have "Items" and "XP" inconsistently translated.

**Comment by Sven792** [comment]
Hmm... no don't merge it. :P

**Comment by Sven792** [comment]
Hmm... XP will be replaced with EP for "**E**rfahrungs**p**unkte" this will be correct.

**Comment by Sven792** [comment]
The word "Item" will be used because there is not much space in the most case. For the description we can use the word "Gegenstand".

**Comment by Sven792** [comment]
Does this pull request have a benefit or should it be closed forever?
